weed/s3api/s3api_server.go

```diff
@@ -76,7 +81,7 @@ func NewS3ApiServer(router *mux.Router, option *S3ApiServerOption) (s3ApiServer
 
 	s3ApiServer.registerRouter(router)
 
-	go s3ApiServer.subscribeMetaEvents("s3", filer.DirectoryEtcRoot, time.Now().UnixNano())
+	go s3ApiServer.subscribeMetaEvents("s3", time.Now().UnixNano(), filer.DirectoryEtcRoot, option.BucketsPath)
```

Comment:
This will listen to all the changes under buckets folder, which will be unnecessarily too much.

Reply:
Yes, you are right, I didn't notice this, I meant to only subscribe to the events of the bucket, not the events of all the files in the bucket

Reply:
If the filer's Notification Queue write interface is open to the s3 Api Server (through grpc), then these s3-related path subscriptions are very flexible. Now I need to subscribe to bucket-related operations (such as create, delete, update acl, etc.), or Subscriptions of other operations (such as bucket policy) do not need to modify the code of the filer.

How about this ? @chrislusf

Reply:
added https://github.com/seaweedfs/seaweedfs/commit/31922b2bf232af8de55adde44138437aeefea1c8 to watch specific directories